### ISSUE TITLE
Is there a way to set MaxCallRecvMsgSize?

### ISSUE THREAD

**Issue by bxq2011hust** [tikv/grpc-rs#602]
I use tikv rust client and got this error `RpcFailure: 8-RESOURCE_EXHAUSTED Received message larger than max (4563651 vs. 4194304)`. From [here](https://stackoverflow.com/questions/55362342/grpc-received-message-larger-than-max-8653851-vs-4194304) I found there is an `MaxCallRecvMsgSize` option in grpc, but I can't find a method to set it in  [grpcio](https://docs.rs/grpcio/latest/grpcio/index.html)::[CallOption](https://docs.rs/grpcio/latest/grpcio/struct.CallOption.html#)

**Comment by BusyJay** [comment]
Yes, check https://docs.rs/grpcio/latest/grpcio/struct.ChannelBuilder.html#method.max_receive_message_len.
